docs/source/support/release-notes.rst

```diff
@@ -6,6 +6,25 @@ Release notes
 
 This page contains release notes for the SDK.
 
+0.12.0
+------
+
+- Change in how values are addressed in Navigator's `frontend-config.js`.
+  
+  - Old syntax for accessing values: `argument.foo.bar`
+  - New syntax: 
+
+    .. code-block:: javascript
+
+        argument.fields.find(f => f.label === "foo").fields.find(f => f.label === "bar").value
```

Comment:
I would not mention the first line here, I don't think people will want to use that. The exact types (their TypeScript definitions) are described if you open `/config` in Navigator, btw.

I would do something like this (note the import statement):
```
    .. code-block:: javascript
        import { DamlLfValue } from '@da/ui-core';
        // Accessing field 'bar' of field 'foo' of the argument
        DamlLfValue.evalPath(argument, ["foo", "bar"])
        DamlLfValue.toJSON(argument).foo.bar
```